cli/bpmetadata/proto/bpmetadata.proto

```diff
@@ -0,0 +1,343 @@
+syntax = "proto3";
+
+package google.cloud.config.bpmetadata;
+
+import "bpmetadata_ui.proto";
+
+option go_package = "github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata";
+
+message BlueprintMetadataSpec {
+
+  // BlueprintInfo defines the basic information of the blueprint.
+  BlueprintInfo info = 1;
+
+  // BlueprintContent defines the detail for blueprint related content such as
+  // related documentation, diagrams, examples etc.
+	BlueprintContent content = 2;
```

Comment:
nit: lets keep these aligned
```suggestion
  BlueprintInfo info = 1;

  // BlueprintContent defines the detail for blueprint related content such as
  // related documentation, diagrams, examples etc.
  BlueprintContent content = 2;
```

Reply:
This is weird. There's no misalignment locally.

Reply:
I think there's a bug in GitHub viewer. It fails to recognize a tab in Go and proto as same. Anyway, I redid it and its fixed now.